crates/notion-core/src/catalog.rs

```diff
@@ -136,6 +146,56 @@ impl Catalog {
 
         Ok(())
     }
+
+    // TODO: refactor activate, install and uninstall methods
```

Comment:
This would be good, agreed -- but in the interest of merging this PR, why don't you file it as an issue and then turn this into an `// ISSUE(#nn): etc` comment.

Reply:
Done https://github.com/notion-cli/notion/issues/87